XamlStyler.Mac/Properties/AddinInfo.cs

```diff
@@ -1,7 +1,7 @@
 ﻿using Mono.Addins;
 using Mono.Addins.Description;
 
-[assembly: Addin("XamlStyler", Namespace = "Xavalon", Version = "1.1.5")]
+[assembly: Addin("XamlStyler", Namespace = "Xavalon", Version = "3.2001.0")]
```

Comment:
Can we continue with the old versioning scheme here for now? Technically, 3.2001.0 already shipped. The goal of #231 is to automate versioning and support simultaneous releases.

Reply:
Okay, I've changed it to 2.0.0, it has some breaking changes, including renaming of global options file storage so it's major update